creator/decorators.py

```diff
@@ -182,11 +223,12 @@ def _get_existing_job_log(self):
         Otherwise, return None if we cannot resolve an existing job log.
         """
 
-        if self._release and self._release.job_log is not None:
-            return self._release.job_log
-
-        if self._task and self._task.job_log is not None:
-            return self._task.job_log
+        # Grab the first job log we find and return it
+        # This could have unforseen implications if the related instances have
+        # different job logs: only the first's log contents will be appended to
+        for instance in self.related_instances:
```

Comment:
🤔 Don't you want to update all related instances' job logs, not just the first one you find?

Reply:
yeah that's probably the way to go.